tsfile/src/test/java/org/apache/iotdb/tsfile/utils/TsFileGeneratorForTest.java

```diff
@@ -159,10 +159,33 @@ static public void write() throws IOException, InterruptedException, WriteProces
     innerWriter = new TsFileWriter(file, schema, TSFileDescriptor.getInstance().getConfig());
 
```

Comment:
I observed that file just now... this should be somewhat similar to the class I introduced: tsfile/src/main/java/org/apache/iotdb/tsfile/write/writer/IncompleteFileTestUtil.java
Should we merge this or something?
Or am I wrong with my assumption?

Reply:
indeed, we can merge this two classes.